app/serializers/api/v3/statement_serializer.rb

```diff
@@ -0,0 +1,18 @@
+module API
+  module V3
+    class StatementSerializer < Blueprinter::Base
+      identifier :id
+
+      field :month
+      field :year
+      field :cohort_id, name: :cohort
+      field :deadline_date, name: :cut_off_date, datetime_format: "%Y-%m-%d"
+      field :payment_date, datetime_format: "%Y-%m-%d"
+      field :created_at, datetime_format: ->(dt) { dt.iso8601 }
+      field :updated_at, datetime_format: ->(dt) { dt.iso8601 }
+
+      field(:paid) { |s, _| s.payment_date.present? }
+      field(:type) { "npq" }
```

Comment:
I think we will not be carrying this over to NPQ as it's an ECF specific field, could we remove it please?